sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -302,9 +301,7 @@ def _init_coef(self, fan_in, fan_out, rng):
             # Use the initialization method recommended by
             # Glorot et al.
```

Comment:
Not relevant to this PR, but do you think adding the paper's name here will make this comment more clear?

I guess it refers to **Understanding the difficulty of training deep feedforward neural networks.**